module/VuFind/src/VuFind/Controller/Plugin/Factory.php

```diff
@@ -210,4 +210,21 @@ public static function getStorageRetrievalRequests(ServiceManager $sm)
             $sm->get('Zend\Session\SessionManager')
         );
     }
+    
+        /**
+     * Construct the StorageRetrievalRequests plugin.
+     *
+     * @param ServiceManager $sm Service manager.
+     *
+     * @return StorageRetrievalRequests
+     */
+    public static function getOverdriveConnector(ServiceManager $sm)
```

Comment:
Is this a factory for something from an earlier design that is no longer needed?

Reply:
I guess it must be.  Will fix in the next commit.